packages/terra-overlay/src/Overlay.jsx

```diff
@@ -106,6 +107,10 @@ class Overlay extends React.Component {
       for (let i = 0; i < this.containerChildren.length; i += 1) {
         prevTabIndex.push(this.containerChildren[i].tabIndex);
         this.containerChildren[i].tabIndex = -1;
+        // childern with calss name Overlay is the overlay component and it's content which should not be disabled for screen readers.
```

Comment:
nit: Change `calss name` to `class name`

Reply:
Thank you : corrected the [typo]([url](https://github.com/cerner/terra-core/pull/2249/commits/9dee2d1f607a345680c9525867030403e74fbee6))